[2025-01-17T10:31:58.081+0000] {processor.py:186} INFO - Started process (PID=70) to work on /opt/airflow/dags/hw_dag.py
[2025-01-17T10:31:58.084+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2025-01-17T10:31:58.090+0000] {logging_mixin.py:190} INFO - [2025-01-17T10:31:58.089+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2025-01-17T10:31:58.203+0000] {logging_mixin.py:190} INFO - [2025-01-17T10:31:58.181+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 16, in <module>
    from modules.pipeline import pipeline
  File "/opt/airflow/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2025-01-17T10:31:58.210+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2025-01-17T10:31:58.240+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.178 seconds
[2025-01-17T10:32:28.594+0000] {processor.py:186} INFO - Started process (PID=134) to work on /opt/airflow/dags/hw_dag.py
[2025-01-17T10:32:28.596+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2025-01-17T10:32:28.599+0000] {logging_mixin.py:190} INFO - [2025-01-17T10:32:28.598+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2025-01-17T10:32:28.685+0000] {logging_mixin.py:190} INFO - [2025-01-17T10:32:28.668+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 16, in <module>
    from modules.pipeline import pipeline
  File "/opt/airflow/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2025-01-17T10:32:28.690+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2025-01-17T10:32:28.722+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.137 seconds
[2025-01-17T10:32:59.302+0000] {processor.py:186} INFO - Started process (PID=198) to work on /opt/airflow/dags/hw_dag.py
[2025-01-17T10:32:59.305+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2025-01-17T10:32:59.315+0000] {logging_mixin.py:190} INFO - [2025-01-17T10:32:59.314+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2025-01-17T10:32:59.430+0000] {logging_mixin.py:190} INFO - [2025-01-17T10:32:59.411+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 16, in <module>
    from modules.pipeline import pipeline
  File "/opt/airflow/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2025-01-17T10:32:59.438+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2025-01-17T10:32:59.473+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.195 seconds
[2025-01-17T10:33:29.555+0000] {processor.py:186} INFO - Started process (PID=262) to work on /opt/airflow/dags/hw_dag.py
[2025-01-17T10:33:29.557+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2025-01-17T10:33:29.563+0000] {logging_mixin.py:190} INFO - [2025-01-17T10:33:29.562+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2025-01-17T10:33:29.644+0000] {logging_mixin.py:190} INFO - [2025-01-17T10:33:29.629+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 16, in <module>
    from modules.pipeline import pipeline
  File "/opt/airflow/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2025-01-17T10:33:29.650+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2025-01-17T10:33:29.698+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.152 seconds
[2025-01-17T10:34:00.650+0000] {processor.py:186} INFO - Started process (PID=326) to work on /opt/airflow/dags/hw_dag.py
[2025-01-17T10:34:00.653+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2025-01-17T10:34:00.663+0000] {logging_mixin.py:190} INFO - [2025-01-17T10:34:00.662+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2025-01-17T10:34:00.857+0000] {logging_mixin.py:190} INFO - [2025-01-17T10:34:00.841+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 16, in <module>
    from modules.pipeline import pipeline
  File "/opt/airflow/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2025-01-17T10:34:00.864+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2025-01-17T10:34:00.893+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.255 seconds
